### DISCUSSION TITLE
mspider/best-practice/optimize-sidecar-resource

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2929] [Announcements]
# mspider/best-practice/optimize-sidecar-resource

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/best-practice/optimize-sidecar-resource.html

**Comment by hanxiaop** [comment]
sidecar现在已经有相关功能了，可以加上

**Comment by nicole-lihui** [comment]
建议添加提示，如果开启命名空间限制，默认开启 istio-system , insight-system ，否则会存在一些问题。
